docs/about.md

```diff
@@ -29,25 +21,23 @@ need to tell a database to flush its in-memory buffers to disk before taking a s
 
 Note that cluster backups are not strictly atomic. If Kubernetes objects are being created or edited at the time of backup, they might not be included in the backup. The odds of capturing inconsistent information are low, but it is possible.
 
-### Scheduled backups
+## Scheduled backups
 
 The **schedule** operation allows you to back up your data at recurring intervals. The first backup is performed when the schedule is first created, and subsequent backups happen at the schedule's specified interval. These intervals are specified by a Cron expression.
 
-A Schedule acts as a wrapper for Backups; when triggered, it creates them behind the scenes.
-
 Scheduled backups are saved with the name `<SCHEDULE NAME>-<TIMESTAMP>`, where `<TIMESTAMP>` is formatted as *YYYYMMDDhhmmss*.
 
-### Restores
+## Restores
 
-The **restore** operation allows you to restore all of the objects and persistent volumes from a previously created Backup. Heptio Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the ones in "123" under "456".
+The **restore** operation allows you to restore all of the objects and persistent volumes from a previously created backup. Ark supports multiple namespace remapping--for example, in a single restore, objects in namespace "abc" can be recreated under namespace "def", and the objects in namespace "123" under "456".
```

Comment:
Maybe worthwhile stating that you can restore all, **or a filtered subset of**, objects/PVs from a previously created backup?